docs/dapps/nfts/preparing-nft-files.md

```diff
@@ -0,0 +1,88 @@
+
+If you right-click on the image, you can copy the image’s url. This URL is important. Copy this down to use in the next step as we set up the metadata. For this example, my URL is `https://gateway.pinata.cloud/ipfs/QmPWbixyMsaNkR9v612bBFbncKGmgXDKz9CgMtDDD7Bymw/0.png`
+
+## Preparing the metadata
+
+Now that we have the image uploaded and its URL, we can create the matching metadata file for it. 
+
+Where this NFT is going to be an ERC721, we know we can use metadata standards often found on Marketplaces such as Joepegs.com. The .json file below is an example of what the [metadata](https://docs.opensea.io/docs/metadata-standards#metadata-structure) should look like.
+
+```json
+{
+    "name": "",
+    "tokenId": 0,
+    "image": "",
+    "description": "",
+    "attributes": []
+}
+```
+
+Now, we will populate the values into the metadata file. You can choose any `name` and `description` that you want.
+
+The `tokenId` here will be `0` so that it corresponds to the image we just uploaded. If uploading multiple files, this needs to be incremented in each file.
+
+The `image` link is the URL we saved from the last step of the previous section. Paste that link here so the smart contract knows where to find the image file for your NFT. If uploading multiple files, the end of the URL (the specific image) needs to increment in each file.
+
+The `attributes` field is not quite as important here, but if you were uploading NFTs with several layers, the attributes would be the information of those specific layers. This is often used when calculating the rarity of NFTs to be able to rank them by how frequently their layers appear throughout the entire collection. It will be erased in this tutorial.
+
+Below is an example of how you would fill out the fields in the metadata file.
+
+```json
+{
+    "name": "Cool Photography",
+    "tokenId": 0,
+    "image": "https://gateway.pinata.cloud/ipfs/QmPWbixyMsaNkR9v612bBFbncKGmgXDKz9CgMtDDD7Bymw/0.png",
+    "description": "A cool image"
+}
+```
+
+When saving this file, you want it to share the same name as the image it corresponds to. In this case, it is 0.
+
+Once the metadata file is uploaded to Pinata, the file extension will actually not be needed. It will search for the file as a directory and be able to pull its information from there. To remove the file extension, follow these steps for a [Mac](https://support.apple.com/guide/mac-help/show-or-hide-filename-extensions-on-mac-mchlp2304/mac) environment, or these for a [Windows](https://www.techwalla.com/articles/how-to-remove-file-extensions) environment.
+
+Now that the file extension has been removed, place it in another folder as you did with the image file. They need to be SEPARATE folders.
+
+![Metadata Folder](preparing-nft-files/7-metadata-folder.png)
+
+You will now repeat the folder upload process to add the metadata to Pinata. Follow the same steps as above. Once completed, you will have both folders available on your dashboard.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "You'll now repeat..." instead of "You will now repeat..." and "you'll have both folders..." instead of "you will have both folders..."